pkg/configuration/configuration_test.go

```diff
@@ -47,6 +51,64 @@ func TestGetAllMemberParameters(t *testing.T) {
 	})
 }
 
+func TestLoadFromConfigMap(t *testing.T) {
+	restore := test.SetEnvVarAndRestore(t, "WATCH_NAMESPACE", "toolchain-member-operator")
+	defer restore()
+	t.Run("default", func(t *testing.T) {
+		// when
+		config := getDefaultConfiguration(t)
+
+		// then
+		assert.Equal(t, "rhd", config.GetIdP())
+	})
+	t.Run("env overwrite", func(t *testing.T) {
+		// given
+		restore := test.SetEnvVarAndRestore(t, "MEMBER_OPERATOR_CONFIG_MAP_NAME", "test-config")
```

Comment:
we should set and restore also the variables retrieved from the ConfigMap:
```suggestion
		restore := test.SetEnvVarsAndRestore(t,
			test.Env("MEMBER_OPERATOR_CONFIG_MAP_NAME", "test-config"),
			test.Env("MEMBER_OPERATOR_IDENTITY_PROVIDER", ""),
			test.Env("MEMBER_OPERATOR_TEST_TEST", ""))
```

Reply:
fixed here: 8a2e7be9cb97a69f7afb5490c1ab47e251946fda